extensions/spring-xd-extension-cassandra/src/main/java/org/springframework/xd/cassandra/mixins/CassandraConnectionMixin.java

```diff
@@ -44,7 +44,7 @@
 
 	private String initScript;
 
-	private String[] entityBasePackages = new String[0];
+	private String[] entityBasePackages;
```

Comment:
@ilaya, maybe we should change the doc generation (eg to use
ArrayUtils.toCommaDelimitedString) because an array will *never* produce
something pretty anyway. So even if we workaround this one, another case
will pop up tomorrow

On Tue, Sep 29, 2015 at 5:22 PM, Artem Bilan <notifications@github.com>
wrote:

> In
> extensions/spring-xd-extension-cassandra/src/main/java/org/springframework/xd/cassandra/mixins/CassandraConnectionMixin.java
> <https://github.com/spring-projects/spring-xd/pull/1785#discussion_r40685793>
> :
>
> > @@ -44,7 +44,7 @@
> >
> >  	private String initScript;
> >
> > -	private String[] entityBasePackages = new String[0];
> > +	private String[] entityBasePackages;
>
> @ilayaperumalg <https://github.com/ilayaperumalg>, have you tested it
> before pushing?
> The reason for the new String[0] is here (AbstractCassandraConfiguration):
>
> /**
>  * The base packages to scan for entities annotated with {@link Table} annotations. By default, returns the package
>  * name of {@literal this} (<code>this.getClass().getPackage().getName()</code>). This method must never return null.
>  */
> public String[] getEntityBasePackages() {
>     return new String[] { getClass().getPackage().getName() };
> }
>
> And so on up to the:
>
>   ArrayList(E[] array) {
>          a = Objects.requireNonNull(array);
> }
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spring-projects/spring-xd/pull/1785/files#r40685793>.
>